.github/workflows/pull_request_checks.yml

```
@@ -0,0 +1,92 @@
+name: Pull Request Checks
+
+on:
+  pull_request:
+
+jobs:
+  testApiAndSourceSets:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.9)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)